package.json

```
@@ -28,13 +28,16 @@
     ]
   },
   "devDependencies": {
+    "@changesets/changelog-github": "^0.5.1",
```

Biome (2.1.2)

[error] The key @changesets/changelog-github was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)